core/management/generate_data.py

```diff
@@ -0,0 +1,83 @@
+from django.core.management import BaseCommand
+
+from spaces.models import Space
+from posts.models import Post
+from users.models import User
+from users.tests.factories import UserFactory
+
+class Command(BaseCommand):
+    help = "Creates a random local data for testing."
+
+    def handle(self, *args, **kwargs):
+        for _ in range(10):
+            UserFactory()
+
+        spaces = [
+            Space(name="Cat Memes"),
+            Space(name="Tech Geeks"),
+            Space(name="Pizza Lovers"),
+            Space(name="Coffee Addicts"),
+        ]
+        Space.objects.bulk_create(spaces)
+
+        print("Mock data created.")
+
+for space in spaces:
+    space.save()
+
+for _ in range(4):  
+    user = User(
+        email=fake.email(),
```

Comment:
Fake library is not included in imports. You can import them here or can use UserFactory to generate random users here.